tyk-api/tyk_api.go

```diff
@@ -35,14 +37,23 @@ type EndpointConfig struct {
 }
 
 type OAuthResponse struct {
-	AccessToken string `json:"access_token"`
-	ExpiresIn   int64  `json:"expires_in"`
-	RedirectTo  string `json:"redirect_to"`
-	TokenType   string `json:"token_type"`
+	AccessToken DBString `json:"access_token"`
+	ExpiresIn   int64    `json:"expires_in"`
+	RedirectTo  string   `json:"redirect_to"`
+	TokenType   string   `json:"token_type"`
+}
+
+type DBString string
+
+func (DBString) DBID() apidef.ObjectId {
```

Comment:
Same here, why it should be a DBString type, and interface?

Reply:
`SetKey` is called with `AccessKey` and to make the code compile, it implements the `DBObject` interface.

Reply:
https://github.com/TykTechnologies/tyk-identity-broker/blob/e623d20c9c04aba7d1e60fcee944dacaa5614968/tap/identity-handlers/tyk_handler.go#L377

Reply:
How it gets saved in the SQL layer?

Reply:
Honestly, I am not sure. How it was stored previously and now needs checking.